src/main/java/io/jenkins/plugins/leanixmi/LeanIXLogAction.java

```diff
@@ -24,6 +25,10 @@
     public static final String VERSION_NOTSET = "Deployment version variable or deployment version value is not set in jenkins environment.";
     public static final String DEPENDENCIES_NOT_GENERATED = "The dependencies of your project couldn't be generated.";
     public static final String DEPENDENCY_MANAGER_NOT_SET = "The dependencies of your project couldn't be generated. The dependency manager doesn't seem to be set correctly. Please check your settings.";
+    public static final String MAVEN_SETTINGS_FILE_WRONG = "The maven settings file could not be parsed, please check that it is correct.";
+    public static final String MAVEN_SETTINGS_FILE_NOTFOUND =
+        "The maven settings file could not be found in your Source Code Management "
+            + "System, please check that the path you specified is correct.";
```

Comment:
Can you add another log statement in case maven setting path is present. similar to https://github.com/leanix/github-actions-store-leanix-plugin/blob/c7c042cfe9605a0fb62c661a7d259eb8193cdacd/entrypoint.sh#L9

Reply:
Included the logger stmts as here - https://github.com/jenkinsci/leanix-microservice-intelligence-plugin/pull/21/files#diff-54d89e065ba6adaec80abbe361f065aa6a815ae840cac4c4199777d4f285b802R92-R102